packages/ra-i18n-i18next/README.md

```diff
@@ -0,0 +1,221 @@
+
+```tsx
+import { Admin } from 'react-admin';
+import { useI18nextProvider } from 'ra-i18n-i18next';
+import i18n from 'i18next';
+import resourcesToBackend from 'i18next-resources-to-backend';
+
+const App = () => {
+    const i18nextInstance = i18n.use(
+        // Here we use a Backend provided by i18next that allows us to load
+        // the translations however we want.
+        // See https://www.i18next.com/how-to/add-or-load-translations#lazy-load-in-memory-translations
+        resourcesToBackend(language => {
+            if (language === 'fr') {
+                // Load the ra-language-french package and convert its translations in i18next format
+                return import(
+                    `ra-language-french`
+                ).then(({ default: messages }) =>
+                    convertRaTranslationsToI18next(messages)
+                );
+            }
+            // Load the ra-language-english package and convert its translations in i18next format
+            return import(`ra-language-english`).then(({ default: messages }) =>
+                convertRaTranslationsToI18next(messages)
+            );
+        })
+    );
+
+    const i18nProvider = useI18nextProvider({
+        i18nextInstance,
+        availableLocales: [
+            { locale: 'en', name: 'English' },
+            { locale: 'fr', name: 'French' },
+        ],
+    });
+
+    if (!i18nProvider) return (<div>Loading...</div>);
+
+    return (
+        <Admin i18nProvider={i18nProvider}>
+           ...
+        </Admin>
+    );
+};
+```
+
+### `convertRaMessagesToI18next` function
+
+A function that takes translations from a standard react-admin language package and converts them to i18next format.
+It transforms the following:
+- interpolations wrappers from `%{foo}` to `{{foo}}` unless a prefix and/or a suffix are provided
+- pluralization messages from a single key containing text like `"key": "foo |||| bar"` to multiple keys `"foo_one": "foo"` and `"foo_other": "bar"`
+
+#### Usage
+
+```ts
+import englishMessages from 'ra-language-english';
+import { convertRaMessagesToI18next } from 'ra-i18n-18next';
+
+const messages = convertRaMessagesToI18next(englishMessages);
+```
+
+#### Parameters
+
+| Parameter            | Required | Type        | Default | Description                                                      |
+| -------------------- | -------- | ----------- | ------- | ---------------------------------------------------------------- |
+| `raMessages`         | Required | object      |         | An object containing standard react-admin translations such as provided by ra-language-english |
+| `options`            | Optional | object      |         | An object providing custom interpolation suffix and/or suffix |
+
+@example Convert the english translations from ra-language-english to i18next format with custom interpolation wrappers
```

Comment:
`@example` ??

Besides, it would be nice to have an actual example of translation keys before and after calling `convertRaMessagesToI18next`

Reply:
It's explained with examples in the introduction for this function